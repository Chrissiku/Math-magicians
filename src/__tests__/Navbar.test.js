import Navigation from "../components/Navbar";
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from  'react-test-renderer';

it("Testing Navbar component", () => {
    const TREE = renderer.create(
        <Router>
            <Navigation />
        </Router>
    ).toJSON();

    expect(TREE).toMatchSnapshot();
})