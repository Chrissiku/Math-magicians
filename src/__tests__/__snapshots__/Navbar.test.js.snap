// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Navbar component 1`] = `
<header
  className="header"
>
  <a
    className="logo"
    href="/"
    onClick={[Function]}
  >
    Math Magicians
  </a>
  <nav>
    <ul>
      <a
        href="/"
        onClick={[Function]}
      >
        <li>
          <a
            href=""
          >
            Home
          </a>
        </li>
      </a>
      <a
        href="/calculator"
        onClick={[Function]}
      >
        <li>
          <a
            href=""
          >
            Calculator
          </a>
        </li>
      </a>
      <a
        href="/quote"
        onClick={[Function]}
      >
        <li>
          <a
            href=""
          >
            Quote
          </a>
        </li>
      </a>
    </ul>
  </nav>
</header>
`;
